roles/rancher/tasks/main.yml

```diff
@@ -553,6 +553,7 @@
     --name "cloudman"
     --namespace "{{ cm_namespace_name }}"
     --version "{{ cm_helm_version }}"
+    --set galaxy_oidc_client_secret="{{ random_client_secret }}"
```

Comment:
@almahmoud Cloudman is not "aware" of Galaxy, since it's just another helmsman application. Therefore, I think the correct place to set this value would be within the galaxy helm chart. That is, either autogenerate a random value if non-specified, or make helmsman generate and send the value.